validator.go

```diff
@@ -170,15 +170,15 @@ func (v *validator) ValidateHttpRequestWithPathItem(request *http.Request, pathI
 	reqBodyValidator := v.requestValidator
 
 	// create some channels to handle async validation
-	doneChan := make(chan bool)
+	doneChan := make(chan struct{})
```

Comment:
I don't understand why? we only need a bit, we don't need a whole struct?

Reply:
No memory consumption,`struct{}` is the smallest data type available in Go